Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -294,6 +294,12 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate, UITextFieldDele
         if editTextField.text.isEmpty {
             editTextField.text = clearedText ?? ""
         }
+
+        if let url = self.url {
```

Comment:
I think this should be an else if otherwise you'll just overwrite what we've just done